Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.scss

```diff
@@ -70,6 +70,11 @@ mat-sidenav-container {
   }
 }
 
+.menu {

```

Comment:
The class is not used?

Reply:
https://github.com/T-Systems-MMS/phonebook/pull/376/files#diff-3a2349a260d775da2f802c9de2cfddd8
the menu class is used in line 128